package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "oci-metrics-datasource",
   "private": true,
-  "version": "3.0.5",
+  "version": "3.0.6",
```

Comment:
There is also a plugin version number in the `src/plugin.json` file

Reply:
good, modified now